Widgets.go

```diff
@@ -2379,3 +2378,48 @@ func (d *DatePickerWidget) Build() {
 		imgui.PopID()
 	}
 }
+
+type ColorEditWidget struct {
+	label    string
+	color    *color.RGBA
```

Comment:
@AllenDang I'm not sure, is it a okey, or I should put imgui.Vec4 here?
in fact, it makes a lot of conversions later, but is easier in use (in my view)